software/dittofeed.yml

```
@@ -0,0 +1,13 @@
+name: "Dittofeed"
+website_url: "https://www.dittofeed.com"
+source_code_url: "https://github.com/dittofeed/dittofeed"
+description: "Omni-channel customer engagement and messaging automation platform. Alternative to Braze, Customer.io, and Iterable."
+licenses:
+  - MIT
+platforms:
+  - Typescript
```

```suggestion
  - Javascript
```
Javascript is essentially the same as typescript, so this platform tag might work fine, feel free to correct collaborators, otherwise nevermind, as we don't have a typescript tag.